src/services/measure/measure.js

```diff
@@ -0,0 +1,270 @@
+goog.require('ol.interaction.Pointer');
+goog.require('ol.style.Style');
+
+
+
+/**
+ * @classdesc
+ * Interaction that allows measuring (length, area, ...).
+ *
+ * @constructor
+ * @extends {ol.interaction.Pointer}
+ */
+ngeo.interaction.Measure = function() {
+
+
+  /**
+   * The help tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.helpTooltipElement_;
+
+
+  /**
+   * Overlay to show the help messages.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.helpTooltip_;
+
+
+  /**
+   * The measure tooltip element.
+   * @type {Element}
+   * @private
+   */
+  this.measureTooltipElement_;
+
+
+  /**
+   * Overlay to show the measurement.
+   * @type {ol.Overlay}
+   * @private
+   */
+  this.measureTooltip_;
+
+
+  /**
+   * Pointer move listener key
+   * @type {goog.events.Key}
+   * @private
+   */
+  this.pointerMoveHandlerKey_;
+
+
+  /**
+   * The sketch feature.
+   * @type {ol.Feature}
+   */
+  this.sketchFeature;
+
+  /**
+   * The draw interaction to be used.
+   * @type {ol.interaction.Draw|ngeo.interaction.DrawAzimut}
+   */
+  this.drawInteraction = null;
+
+  goog.base(this);
+
+
+  /**
+   * The draw overlay
+   * @type {ol.FeatureOverlay}
+   */
+  this.overlay = new ol.FeatureOverlay({
+    style: [
+      new ol.style.Style({
+        fill: new ol.style.Fill({
+          color: 'rgba(255, 255, 255, 0.2)'
+        })
+      }),
+      new ol.style.Style({
+        stroke: new ol.style.Stroke({
+          color: 'white',
+          width: 5
+        })
+      }),
+      new ol.style.Style({
+        stroke: new ol.style.Stroke({
+          color: '#ffcc33',
+          width: 3
+        })
+      })
+    ]
+  });
+};
+goog.inherits(ngeo.interaction.Measure, ol.interaction.Pointer);
+
+
+/**
+ * @inheritDoc
+ */
+ngeo.interaction.Measure.prototype.setMap = function(map) {
+  goog.base(this, 'setMap', map);
+  this.overlay.setMap(map);
+
+  map.addInteraction(this.drawInteraction);
+
+  this.drawInteraction.on('drawstart', goog.bind(this.onDrawStart_, this));
+  this.drawInteraction.on('drawend', goog.bind(this.onDrawEnd_, this));
+};
+
+
+/**
+ * Handle draw interaction `drawstart` event.
+ * @param {ol.DrawEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.onDrawStart_ = function(evt) {
+  this.sketchFeature = evt.feature;
+  this.overlay.getFeatures().clear();
+  this.createMeasureTooltip_();
+};
+
+
+/**
+ * Handle draw interaction `drawend` event.
+ * @param {ol.DrawEvent} evt
+ * @private
+ */
+ngeo.interaction.Measure.prototype.onDrawEnd_ = function(evt) {
+  // FIXME we shouldn't manipulate DOM style here
+  this.measureTooltipElement_.className = 'tooltip tooltip-static';
+  this.measureTooltip_.setOffset([0, -7]);
+  this.sketchFeature = null;
+};
+
+
+/**
+ * Creates a new help tooltip
+ * @private
+ */
+ngeo.interaction.Measure.prototype.createHelpTooltip_ = function() {
+  this.removeHelpTooltip_();
+  this.helpTooltipElement_ = document.createElement('div');
```

Comment:
Here you also could use `goog.dom.createDom`, like here: 
https://github.com/openlayers/ol3/blob/master/src/ol/control/overviewmapcontrol.js#L68